(ym) miami progress

YeMyr opened the pull request:
[DEMO LINK](https://yemyr.github.io/layout_miami/)

silchyn reviewed (changes requested):
- Links in `Contact Us` do not change on hover
- The form button has a border
- Maybe we should scroll the user to the top after the form submission

YeMyr commented:
Hi, Ivan! I've made corrections, please, check it
> * Links in `Contact Us` do not change on hover
added animated underline for links
> * The form button has a border
reseted outline and borders
> * Maybe we should scroll the user to the top after the form submission
added to js relocation to the top of page on submiting
